ReactCommon/react/renderer/mapbuffer/MapBuffer.h

```diff
@@ -80,7 +80,8 @@ class MapBuffer {
     uint32_t bufferSize; // Amount of bytes used to store the map in memory
   };
 
-  struct __attribute__((__packed__)) Bucket {
+  #pragma pack(1)
```

Comment:
Should we use #pragma(push, 1) and #pragma(pop) after the strict definitions?

Reply:
The change also broke CI, we should probably reset it after strict definitions :)

Reply:
Will do!